src/appcast.cpp

```diff
@@ -212,10 +213,14 @@ bool is_suitable_windows_item(const Appcast &item)
 
     // Check suffix for matching bitness
 #ifdef _WIN64
+#ifdef __AARCH64EL__ || _M_ARM64
```

Comment:
This is incorrect, it needs to be `#if defined(__AARCH64EL__) || defined(_M_ARM64)`.

Also, where does `__AARCH64EL__ ` come from? Can't find any reference to in relation to Windows...

Reply:
`__AARCH64EL__` is defined by LLVM MinGW:

```
parallels@ubuntu-linux-22-04-desktop:~/winsparkle/build$ aarch64-w64-mingw32-clang -E -dM - </dev/null | grep AARCH64
#define __AARCH64EL__ 1
#define __AARCH64_CMODEL_SMALL__ 1
parallels@ubuntu-linux-22-04-desktop:~/winsparkle/build$
```

Unfortunately it doesn't look like that version of MinGW defines `_M_ARM64`, so we can't simply check for that.